### ISSUE TITLE
Error using  function generate_random_graph

### ISSUE THREAD

**Issue by ibirisol** [djordon/queueing-tool#48]
Hi there,

Using queueing_tool with basic example from [1], following error appear:
```
>>> import queueing_tool as qt
>>> g = qt.generate_random_graph(5, seed=10)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "~/.env/lib/python3.5/site-packages/queueing_tool/graph/graph_generation.py", line 121, in generate_random_graph
    g = minimal_random_graph(num_vertices, **kwargs)
  File "~/.env/lib/python3.5/site-packages/queueing_tool/graph/graph_generation.py", line 225, in minimal_random_graph
    g = QueueNetworkDiGraph(g.to_directed())
  File "~/.env/lib/python3.5/site-packages/queueing_tool/graph/graph_wrapper.py", line 243, in __init__
    edges.sort()
AttributeError: 'OutEdgeView' object has no attribute 'sort'
```

[1] http://queueing-tool.readthedocs.io/en/latest/network.html

**Comment by djordon** [comment]
Hey @ibirisol, what version of of queueing tool are you using? That looks like a bug from version 1.2.0 or earlier, but it should be fixed in the latest version v1.2.1.

**Comment by ibirisol** [comment]
Hi @djordon, Thanks!
I'm using pip latest version. That is working from now.
